handout/http/catching-rejections/cancel_request.md

```diff
@@ -0,0 +1,25 @@
+## Cancel a Request
+
+Cancelling an HTTP request is a common requirement. For example, you could have a queue of requests where a new request supersedes a pending request and that pending request needs to be cancelled.
+
+To cancel a request we call the `unsubscribe` function of its subscription.
+
+```ts
+@Component({ ... })
+export class MyApp {
+	...
+
+	search() {
+		const request = this.searchService.search(this.searchField.value)
+		  .subscribe(
+		    (result) => { this.result = result.artists.items; },
+		    (err) => { this.errorMessage = err.message; },
+        () => { console.log('Completed'); }
```

Comment:
If you see this line directly on the gitbook, there seems to be an indentation problem.

![screen shot 2016-11-18 at 4 28 25 pm](https://cloud.githubusercontent.com/assets/2279046/20447291/11dd20f4-adac-11e6-84ce-43be34ba4481.png)